src/payment/strategies/googlepay/googlepay-checkoutcom-initializer.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { InvalidArgumentError } from '../../../common/error/errors';
+
+import GooglePayCheckoutcomInitializer from './googlepay-checkoutcom-initializer';
+import { getCheckoutMock, getGooglePaymentCheckoutcomDataMock, getGooglePaymentDataMock, getGooglePayCheckoutcomPaymentDataRequestMock, getGooglePayTokenizePayloadCheckoutcom, getPaymentMethodMock } from './googlepay.mock';
+
+describe('GooglePayCheckoutcomInitializer', () => {
+    it('creates an instance of GooglePayCheckoutcomInitializer', () => {
```

Comment:
This test may not be necessary at all, as the instance is created manually 🍹

Reply:
removed